crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -137,6 +137,180 @@ pub struct StandardMaterial {
+    ///
+    /// - R - `transmission_texture`
+    /// - G - `thickness_texture`
+    /// - B - _unused_
+    /// - A - `diffuse_transmission_texture`
+    ///
+    /// The `KHR_materials_diffuse_transmission` glTF extension also defines a `diffuseTransmissionColorTexture`,
+    /// that _we don't currently support_. One might choose to pack the intensity and color textures together,
+    /// using RGB for the color and A for the intensity, in which case this packing advice doesn't really apply.
+    #[texture(17)]
+    #[sampler(18)]
+    pub diffuse_transmission_texture: Option<Handle<Image>>,
+
+    /// The amount of light transmitted _specularly_ through the material (i.e. via refraction)
+    ///
+    /// Implemented as a relatively expensive screen-space effect that allows ocluded objects to be seen through the material, while
+    /// taking [`StandardMaterial::ior`], [`StandardMaterial::thickness`] and [`StandardMaterial::perceptual_roughness`] into account. If purely
+    /// diffuse light transmission is needed, (i.e. “translucency”) consider using [`StandardMaterial::diffuse_transmission`] instead.
+    ///
+    /// - When set to `0.0` (the default) no light is transmitted.
+    /// - When set to `1.0` all light is transmitted through the material.
+    ///
+    /// The material's [`StandardMaterial::base_color`] also modulates the transmitted light.
+    ///
+    /// **Note:** Typically used in conjunction with [`StandardMaterial::thickness`] and [`StandardMaterial::ior`].
+    #[doc(alias = "refraction")]
+    pub transmission: f32,
+
+    /// A map that modulates specular transmission via its **R channel**. Multiplied by [`StandardMaterial::transmission`]
+    /// to obtain the final result.
+    ///
+    /// **Important:** The [`StandardMaterial::transmission`] property must be set to a value higher than 0.0,
+    /// or this texture won't have any effect.
+    ///
+    /// ## Texture Packing
+    ///
+    /// Due to how channels are mapped, you can more efficiently use a single shared texture image
+    /// for configuring the following:
+    ///
+    /// - R - `transmission_texture`
+    /// - G - `thickness_texture`
+    /// - B - _unused_
+    /// - A - `diffuse_transmission_texture`
+    ///
+    /// The `KHR_materials_diffuse_transmission` glTF extension also defines a `diffuseTransmissionColorTexture`,
+    /// that _we don't currently support_. One might choose to pack the intensity and color textures together,
+    /// using RGB for the color and A for the intensity, in which case this packing advice doesn't really apply.
+    #[texture(13)]
+    #[sampler(14)]
+    pub transmission_texture: Option<Handle<Image>>,
+
+    /// Thickness of the volume beneath the material surface.
+    ///
+    /// When set to `0.0` (the default) the material appears as an infinitely-thin film,
+    /// transmitting light without distorting it.
+    ///
+    /// When set to any other value, the material distorts light like a volumetric lens.
```

Comment:
```suggestion
    /// When set to any other value, the material distorts light, e.g. a thick crystal ball.
```

My wording is terrible, but lets include a real-life example here.

Reply:
I tentatively went with an in between of the two, “like a thick lens”.